### ISSUE TITLE
fundmasternode -> "Transaction is not valid" on MN setup

### ISSUE THREAD

**Issue by JeffPrixla1986** [Divicoin/Divi#48]
# Expected behavior
After fundmasternode we should have the "config line"

# Actual behavior
We have below error:

`error: {"code":-25,"message":"Transaction is not valid"}`

# Operating system
Place an 'x' in the box next to your operating system
* [ ] Mac OSX 10.13.1
* [ ] Linux Ubuntu 16.04.4
* [ ] Linux Ubuntu 17.04.4
* [ ] Linux Ubuntu 18.04.4
* [x] Debian 9.4
* [ ] Windows 10

# Steps to reproduce the problem
Follow guide at https://blog.diviproject.org/p/2b448d7d-9276-4c79-9875-3ee7431fbcd7/

**Comment by gingfinger** [comment]
It seems to be something wrong with the amount of funds it allocated when selecting a tier..When allocating copper it allocates 0.001divi to the node. I just tried a silver too and it allocated 0.003. Which is in proportion to the 0.001 it did for copper. I don't have enough funds for a true silver, so it shouldn't have went as far as creating a transaction. Should have told me insufficient funds I'd assume

**Comment by Jeff86** [comment]
Yes, it’s seems so, here is what I have when do a listaccounts:

```
./divi-cli listaccounts
{
    "" : -791005.46871228,
    "alloc->linuxjf" : 0.00100000,
    "linuxjf" : 918338.36583171
}
```

We clearly see that 0.00100000 is allocated instead of 100000.00000000 for the copper.

**Comment by jcmelati** [comment]
It seems that it is allocating 100000 'divi satoshi'.
The correct amount, therefore, has to be set to 10000000000000 'divi satoshi'.

Copper 10000000000000
Silver 30000000000000
Gold 100000000000000
Platinum 300000000000000
Diamond 1000000000000000

**Comment by 99darwin** [comment]
Staged for testing

**Comment by Divicoin** [comment]
Closed in 4.0.7.0
